### DISCUSSION TITLE
Please open new topics at community.penpot.app, the Penpot team no longer participates at Discussions

### DISCUSSION THREAD

**Discussion by myfunnyandy** [uxbox/uxbox#2069] [General]
Hi all!

We've created a new space where you can get in touch with the Penpot core team and the rest of the Community, ask your questions and share your thoughts.

So If you're thinking on posting something here at Discussions, it would be better to do it in the [Penpot Community](https://community.penpot.app/).

See you there!

**Comment by elevenpassin** [comment]
Discourse is a great choice!
